### DISCUSSION TITLE
How to swap one DCS for another

### DISCUSSION THREAD

**Discussion by jgould22** [patroni/patroni#3183] [Q&A] [answered]
I have a small patroni cluster that consists of the following
- 2 postgres machines
- an haproxy load balancing between them
- a consul cluster that is used for the DCS

I would like to swap the existing DCS for another dcs, in this case zookeeper.

Is there any literature or documentation any where that I could use use? 

Is it enough to just copy the cluster state from consul to zookeeper and reload patroni?

**Comment by CyberDem0n** [comment, marked as answer]
1. `patronictl pause`
2. Update config files (patroni.yaml)
3. Restart patroni service on all nodes. Postgres will not be restarted because of `pause`.
4. `patronictl resume`

Patroni will take care of everything. You don't have to copy keys between DCS.

**Reply by jgould22** [reply]
Thanks for the prompt reply, glad to hear it is more simple than I thought!
